zerver/lib/webhooks/git.py

```diff
@@ -86,6 +88,10 @@ def get_pull_request_event_message(
         main_message += '\n' + CONTENT_MESSAGE_TEMPLATE.format(message=message)
     return main_message.rstrip()
 
+def get_ping_message(user_name, integration):
```

Comment:
Hm... I think its good idea to have such event here but I'd rather named it somehow different - `get_setup_webhook_message` (also template name has to be changed.

I think it might be worth to have 2 ways of getting this message - with and without `user_name`